pragmatic.programmer.tips.core/services/TipService.cs

```diff
@@ -55,7 +55,7 @@ public async Task<IEnumerable<Tip>> GetAllTipsAsync()
         {
             await _logger.LogDebug("entered GetAllTipsAsync()");
 
-            var tips = (await _tipsRepository.ReadFromRawTipsTextFile()).ToList();
```

Comment:
it would be nice to do a speed test between the two methods of reading. it could be tested again when the method for getting the data from the website is added.

Reply:
Test Results:
```shell
null@null-windows:$ time tips.from.json/pragmatic.programmer.tips.cli.exe

real    0m3.690s
user    0m0.002s
sys     0m0.000s
null@null-windows:$ time tips.from.text/pragmatic.pr
ogrammer.tips.cli.exe

real    0m2.748s
user    0m0.002s
sys     0m0.000s
null@null-windows:$ time tips.from.json/pragmatic.programmer.tips.cli.exe

real    0m2.854s
user    0m0.000s
sys     0m0.002s
null@null-windows:$ time tips.from.text/pragmatic.programmer.tips.cli.exe

real    0m2.750s
user    0m0.002s
sys     0m0.000s
null@null-windows:$ time tips.from.json/pragmatic.programmer.tips.cli.exe

real    0m2.500s
user    0m0.001s
sys     0m0.000s
null@null-windows:$
```

Reply:
A better way of testing would be to use Unit Tests in Visual Studio since that has a built-in timer. I will test again once read-from-website-source implementation is added.